packages/rum-core/test/common/event-target-patch.spec.js

```diff
@@ -151,26 +151,6 @@ describe('EventTargetPatch', function () {
         expect(count).toBe(4)
         expect(events.length).toBe(8)
       })
-
-      it('should not instrument non-Element targets', () => {
```

Comment:
This is not required anymore since from now on the **click** will be also handled in document and window